OG_Format.adoc

```diff
@@ -133,8 +133,8 @@ Name of the platform(s) that supported the sensors data used to create this data
 
 https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/collection/L06/current/_]
 
- |mandatory |“Autonomous Underwater Vehicle”
-|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
+ |mandatory |"sub-surface gliders"
+|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/25/_]
```

Comment:
I just noticed an inconsistency in the text vs link. Is it 27 or 25?

Reply:
Hum, nice. Thanks.
Yes this is 27... le right vocab is "sub-surface gliders".